src/storage/src/lib.rs

```diff
@@ -41,6 +41,7 @@
 #![feature(exact_size_is_empty)]
 #![feature(once_cell)]
 #![cfg_attr(coverage, feature(no_coverage))]
+#![recursion_limit = "256"]
```

Comment:
Why do we change `recursion_limit`? Is it because Introducing OpenTelemetry tracing will cause us to exceed the default `resursion_limit` (128)? If so, will it significantly increase the compile time?

Reply:
Yes, with the changes in this PR, the default `recursion_limit` will be exceeded. But I'm not sure which component exceeded it. I guess it might be `bcc`.

Reply:
I tried again. It's not caused by `bcc`. 🥲